.travis.yml

```diff
@@ -6,11 +6,13 @@ cache:
   npm: false
   yarn: true
 install:
-  - sudo apt remove -y --purge lxd lxd-client
-  - sudo snap install lxd
+  #- sudo apt remove -y --purge lxd lxd-client
+  - sudo snap install lxd --channel=3.0/stable
```

Comment:
why would I fix lxd to a particular version with the implied legacy of having to update it regularly and certainly of the scope of this issue titled "spelling, sentence structure and vocabulary suggestions"